<ui version="4.0" >
 <class>GasRegister</class>
 <widget class="QDialog" name="GasRegister" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Register GasMan</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="nameLineEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Company:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="companyLineEdit" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>License Key:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="keyLineEdit" />
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="infoLineEdit" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Information</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Register using key file</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbTemp" >
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openFile" >
        <property name="text" >
         <string>&amp;Open File...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>companyLineEdit</tabstop>
  <tabstop>infoLineEdit</tabstop>
  <tabstop>keyLineEdit</tabstop>
  <tabstop>openFile</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>pbTemp</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GasRegister</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
